traffic_ops/app/lib/UI/ConfigFiles.pm

```diff
@@ -160,6 +160,37 @@ sub server_data {
 	return $server;
 }
 
+sub get_ats_major_version {
```

Comment:
This appears to be copy-pasted. Is there no reasonable way to use the same function?